operator/src/main/java/org/keycloak/operator/v2alpha1/crds/KeycloakRealmImportSpec.java

```diff
@@ -16,15 +16,18 @@
  */
 package org.keycloak.operator.v2alpha1.crds;
 
+import com.fasterxml.jackson.annotation.JsonPropertyDescription;
 import org.keycloak.representations.idm.RealmRepresentation;
 
 import javax.validation.constraints.NotNull;
 
-public class RealmSpec {
+public class KeycloakRealmImportSpec {
 
     @NotNull
+    @JsonPropertyDescription("The name of the Keycloak CR to reference, in the same namespace.")
```

Comment:
Is the system capable of handling several Keycloaks in the same namespace ? As I see the deployment that's not possible.
So, should the KeycloakCR auto discovered ?

Reply:
Theoretically, it should  be possible to run several KCs in the same namespace.